Broken CD build results in outdated api.stackstorm.com

This repository is responsible for generating https://api.stackstorm.com/ for stable st2 version and https://api-unstable.stackstorm.com/  for st2 `master` based on StackStorm/st2 Swagger spec.

Example build: https://circleci.com/gh/StackStorm/st2apidocs/516
Failure happens in "Generate API Documentation" step:
```
#!/bin/bash -eo pipefail
if [[ "${ST2_BRANCH}" =~ ^v[0-9]+\.[0-9]+$ ]]; then
  st2apidocgen --render --output
elif [ "${ST2_BRANCH}" == "master" ]; then
  st2apidocgen --render --output
else
  st2apidocgen --render /${CIRCLE_BUILD_NUM} --output
fi

(node:195) UnhandledPromiseRejectionWarning: Error: Cannot find module '@babel/core'
 babelify@10 requires Babel 7.x (the package '@babel/core'). If you'd like to use Babel 6.x ('babel-core'), you should install 'babelify@8'.
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/usr/local/lib/node_modules/st2apidocgen/node_modules/babelify/index.js:9:11)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
(node:195) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:195) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

which is related to Javascript dependencies for this repository.

The regression happened in https://github.com/StackStorm/st2apidocs/pull/7 and https://github.com/StackStorm/st2apidocgen/pull/6

Ideally to fix the regression and update Javascript dependencies for this repo to still function properly.

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -25,7 +25,7 @@ jobs:
             statuscode=$(curl --silent --write-out "%{http_code}" -o openapi.yaml https://raw.githubusercontent.com/StackStorm/st2/${ST2_BRANCH}/st2common/st2common/openapi.yaml)
             if test $statuscode -ne 200; then
               echo 'openapi.yaml is not available in https://raw.githubusercontent.com/StackStorm/st2/${ST2_BRANCH}/st2common/st2common/openapi.yaml, get the openapi.yaml from master branch'
-              curl -Ss -q -o openapi.yaml https://raw.githubusercontent.com/StackStorm/st2/${ST2_BRANCH}/st2common/st2common/openapi.yaml
+              curl -Ss -q -o openapi.yaml https://raw.githubusercontent.com/StackStorm/st2/master/st2common/st2common/openapi.yaml
             fi
       - run:
           name: Install Dependencies

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -22,7 +22,11 @@ jobs:
       - run:
           name: Fetch latest StackStorm OpenAPI spec
           command: |
-            curl -Ss -q -o openapi.yaml https://raw.githubusercontent.com/StackStorm/st2/${ST2_BRANCH}/st2common/st2common/openapi.yaml
+            statuscode=$(curl --silent --write-out "%{http_code}" -o openapi.yaml https://raw.githubusercontent.com/StackStorm/st2/${ST2_BRANCH}/st2common/st2common/openapi.yaml)
+            if test $statuscode -ne 200; then
+              echo 'openapi.yaml is not available in https://raw.githubusercontent.com/StackStorm/st2/${ST2_BRANCH}/st2common/st2common/openapi.yaml, get the openapi.yaml from master branch'
+              curl -Ss -q -o openapi.yaml https://raw.githubusercontent.com/StackStorm/st2/master/st2common/st2common/openapi.yaml
+            fi
       - run:
           name: Install Dependencies
           command: npm install
@@ -34,7 +38,7 @@ jobs:
           command: npm test
       - run:
           name: Install doc generator
-          command: npm install -g https://github.com/StackStorm/st2apidocgen
+          command: npm install -g https://github.com/Sheshagiri/st2apidocgen#shesh/fix-babel-issue
       - run:
           name: Generate API documentation
           command: |
@@ -57,6 +61,7 @@ jobs:
       - deploy:
           name: Deploying to S3
           command: |
+            echo 'sync to aws $REGION region.'
             find ~/st2apidocs/build | sed 's|[^/]*/|  |g'
             if [[  "${ST2_BRANCH}" =~ ^v[0-9]+\.[0-9]+$ ]]; then
               aws s3 sync build/ s3://api.stackstorm.com/ --region ${REGION} --delete

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -38,7 +38,7 @@ jobs:
           command: npm test
       - run:
           name: Install doc generator
-          command: npm install -g https://github.com/Sheshagiri/st2apidocgen#shesh/fix-babel-issue
+          command: npm install -g https://github.com/StackStorm/st2apidocgen
       - run:
           name: Generate API documentation
           command: |

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -25,7 +25,7 @@ jobs:
             statuscode=$(curl --silent --write-out "%{http_code}" -o openapi.yaml https://raw.githubusercontent.com/StackStorm/st2/${ST2_BRANCH}/st2common/st2common/openapi.yaml)
             if test $statuscode -ne 200; then
               echo 'openapi.yaml is not available in https://raw.githubusercontent.com/StackStorm/st2/${ST2_BRANCH}/st2common/st2common/openapi.yaml, get the openapi.yaml from master branch'
-              curl -Ss -q -o openapi.yaml https://raw.githubusercontent.com/StackStorm/st2/master/st2common/st2common/openapi.yaml
+              curl -Ss -q -o openapi.yaml https://raw.githubusercontent.com/StackStorm/st2/${ST2_BRANCH}/st2common/st2common/openapi.yaml
             fi
       - run:
           name: Install Dependencies

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "st2apidocs",
-  "version": "1.1.0",
+  "version": "1.2.0",
   "description": "Auto-generated StackStorm documentation",
   "main": "index.js",
   "scripts": {
@@ -26,13 +26,14 @@
     "eslint-plugin-react": "^7.14.2"
   },
   "dependencies": {
+    "@babel/core": "7.0.0",
     "@stackstorm/react-sticky": "5.0.5",
     "commonmark": "0.27.0",
     "commonmark-react-renderer": "4.3.4",
-    "http-server": "^0.11.1",
+    "http-server": "^0.12.3",
     "js-yaml": ">=3.13.1",
     "json-schema-ref-parser": "7.1.0",
-    "lodash": "4.17.11",
+    "lodash": "^4.17.15",
     "react": "15.4.2",
     "react-dom": "15.4.2",
     "react-router-dom": "^5.0.1",

